src/components/box/Box.js

```diff
@@ -149,6 +173,8 @@ Box.propTypes = {
 };
 
 Box.defaultProps = {
+  borderColor: 'neutral',
+  borderTint: 'dark',
```

Comment:
Q: Why no borderWidth defaultprop? won't it be `undefinedpx` now?

Reply:
The border styles are only applied when a border width is set, because of these lines:
`...(borderBottomWidth && { borderBottom: this.getBorder(borderBottomWidth) }),`